fastly/resource_fastly_service_acl_entries_v1.go

```diff
@@ -314,6 +302,24 @@ func executeBatchACLOperations(conn *gofastly.Client, serviceID, aclID string, b
 	return nil
 }
 
+func buildBatchACLEntry(v map[string]interface{}, op gofastly.BatchOperation) *gofastly.BatchACLEntry {
+	entry := &gofastly.BatchACLEntry{
+		Operation: op,
+		ID:        gofastly.String(v["id"].(string)),
+		IP:        gofastly.String(v["ip"].(string)),
+		Negated:   gofastly.CBool(v["negated"].(bool)),
+		Comment:   gofastly.String(v["comment"].(string)),
+	}
+
+	subnet := convertSubnetToInt(v["subnet"].(string))
+	// only set zero subnet if the attribute is explicitly set
+	if v["subnet"].(string) == "0" || subnet != 0 {
```

Comment:
Do you think it might be cleaner to move this logic inside `convertSubnetToInt`? Then `convertSubnetToInt` could return `*int`

Reply:
I think I'm okay with the current